src/pages/_document.js

```diff
@@ -6,6 +6,15 @@ export default class DefaultDocument extends Document {
     return (
       <html lang="en">
         <Head>
+          <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+          <link
```

Comment:
Google Web Fonts has a library that we can programmatically include their web fonts, that way we don't need to hard code the fonts inside the header.

Reply:
@douglasgimli which library is that?